kaniko.go

```diff
@@ -17,6 +17,7 @@ type (
 		Target     string   // Docker build target
 		Repo       string   // Docker build repository
 		Labels     []string // Label map
+		SkipTlsVerify	string // Docker skip tls certificate verify for registry
```

Comment:
change to bool and also run `gofmt` on the codebase, which will automatically format the code.  If you are using an IDE (like vscode) the official Go extension should also autoformat on-save.

Reply:
done